ohw22/seattle/index.md

```diff
@@ -19,7 +19,7 @@ All activities will be in [Ocean Sciences Building (OSB) https://uw.edu/maps/#!/
 
 ## Format
 
-In-person with remote option. We have space reserved to watch OHW tutorials together and work on projects. We also plan to have social gatherings outside of the formal meeting space to explore and enjoy the region together.
+In-person. We have space reserved to watch OHW tutorials together and work on projects. We also plan to have social gatherings outside of the formal meeting space to explore and enjoy the region together. Remote option is possible if impacted by COVID-19 (see below) or other unpredictable events.
```

Comment:
Would you like to make this statement a bit *less* open to a remote option? Something like: 

```suggestion
In-person. We have space reserved to watch OHW tutorials together and work on projects. We also plan to have social gatherings outside of the formal meeting space to explore and enjoy the region together. Remote option may be possible if impacted by COVID-19 (see below) or other unpredictable events. Please contact the satellite organizers via Slack.
```

Having a participant attend remotely implies having the Zoom line open and someone monitoring it outside the GVE window, plus other logistical arrangements (sound, video, etc) that may be a burden.